hllplus/hllplus_private_test.go

```diff
@@ -0,0 +1,50 @@
+package hllplus
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("sparseState", func() {
+	var subject *sparseState
+
+	BeforeEach(func() {
+		subject = newSparseState(12, 17, nil)
+	})
+
+	It("should handle duplicates properly", func() {
```

Comment:
that test is good, but you can test it differently (sing public interfaces):

```
It("should estimate sparse (repetitive)",
  subject, _ = hllplus.New(12, 17)
  hashes := make([]uint64, 0, 100)
  for i := 0; i < 100; i++ {
    hashes = append(hashes, rnd.Uint64())
  }

  for i := 0; i < 10; i++ {
    for _, h := range hashes {
      subject.Add(h)
    }
    Expect(subject.Estimate()).To(BeNumerically("==", 100), "cycle: %d", i+1)
  }
})
```

Before your change it would have done this calculation very poorly